### PR TITLE
get execute env from report env

### PR DESCRIPTION
**Pull request by chengjoey**
#### What type of this PR

Add one of the following kinds:
/kind bug

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/bug?id=228040&issueFilter__urlQuery=eyJzdGF0ZUJlbG9uZ3MiOlsiT1BFTiIsIldPUktJTkciLCJXT05URklYIiwiUkVPUEVOIiwiUkVTT0xWRUQiXX0%3D&issueTable__urlQuery=eyJwYWdlTm8iOjEsICJwYWdlU2l6ZSI6MTB9&issueViewGroup__urlQuery=eyJ2YWx1ZSI6InRhYmxlIiwiY2hpbGRyZW5WYWx1ZSI6eyJrYW5iYW4iOiJkZWFkbGluZSJ9fQ%3D%3D&iterationID=506&type=BUG)

#### Specified Reviewers:

/assign @your-reviewer

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |  get execute env from report env            |
| 🇨🇳 中文    |   执行计划增加真实环境的显示           |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by chengjoey** [comment]
specify env:
image

empty env:
image

File: modules/openapi/component-protocol/scenarios/auto-test-plan-detail/components/executeInfo/render.go [hunk 1]
```diff
@@ -310,3 +312,19 @@ func convertReportToConfig(env apistructs.PipelineReport) (apistructs.AutoTestAP
...
+	envByte, err := json.Marshal(env)
+	if err != nil {
+		return ""
+	}
+	configData := apistructs.PipelineReport{}
+	err = json.Unmarshal(envByte, &configData)
+	if err != nil {
+		return ""
+	}
+	return fmt.Sprintf("%v", configData.Meta[autotest.CmsCfgKeyDisplayName])
```
Comment (sfwn) [inline modules/openapi/component-protocol/scenarios/auto-test-plan-detail/components/executeInfo/render.go:329]: how about `Meta` is nil.

**Review by sfwn — CHANGES_REQUESTED** [review]

File: modules/openapi/component-protocol/scenarios/auto-test-plan-detail/components/executeInfo/render.go (hunk 1 above)
Comment (chengjoey) [reply modules/openapi/component-protocol/scenarios/auto-test-plan-detail/components/executeInfo/render.go]: judge autotest.CmsCfgKeyDisplayName is existed in meta
add test if meta is nil

**Review by sfwn — APPROVED** [review]
